tests/test_graphql_api/test_application_section/test_filtering.py

```diff
@@ -1049,17 +1050,19 @@ def test_application_section__filter__by_text_search__application_id(graphql):
     # - There are two applications with one application sections each
     # - A superuser is using the system
     application_1 = ApplicationFactory.create_in_status_draft_no_sections(
+        id=1,
         organisation=None,
         contact_person=None,
         user=None,
     )
     application_2 = ApplicationFactory.create_in_status_draft_no_sections(
+        id=2,
         organisation=None,
         contact_person=None,
         user=None,
     )
-    section_1 = ApplicationSectionFactory.create_in_status_unallocated(application=application_1, name="foo")
-    ApplicationSectionFactory.create_in_status_unallocated(application=application_2, name="bar")
+    section_1 = ApplicationSectionFactory.create_in_status_unallocated(id=3, application=application_1, name="foo")
+    ApplicationSectionFactory.create_in_status_unallocated(id=4, application=application_2, name="bar")
```

Comment:
Just fixing a flaky test here

Reply:
I don't think the hardcoded IDs are needed here, maybe lets just remove them all together

Reply:
They are needed, since application section text search accounts for application ids as well as section ids, so to remove the flakiness we need to give each of them unique ids so that they are not accidentally the same between application and application section (=source of flakiness)